src/cascadia/TerminalSettingsModel/KeyMapping.cpp

```diff
@@ -172,6 +172,8 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
                 // We want to summon the window with the name "_quake" specifically.
                 args->Name(L"_quake");
+                // We want the window to dropdown, with a 200ms duration.
+                args->DropdownDuration(200);

```

Comment:
Hmm... So now I have a question about `"command": "quakeMode"`. If you do something like this:
```js
{ "command": { "action": "quakeMode", "dropdownDuration": 0 } }
```

where does the dropdown duration get set? It seems like we're outright ignoring it.

Reply:
`quakeMode` doesn't accept params. It's literally an explicit specialization of `globalSummon`, literally 

```json
{ 
    "command": { 
        "action": "globalSummon",
        "name": "_quake",
        "toggleVisibility": true,
        "desktop": "toCurrent",
        "dropdownDuration": 200
    }
}
```

If you want quake mode without the dropdown, make it manually with 
```json
{ 
    "command": { 
        "action": "globalSummon",
        "name": "_quake",
        "toggleVisibility": true,
        "desktop": "toCurrent"
    }
}
```

Reply:
I wonder how we gonna present both `globalSummon` and `quakeMode` in SUI, hopefully in an intuitive and educational way.